lighthouse-core/audits/is-on-https.js

```diff
@@ -72,6 +72,11 @@ class HTTPS extends Audit {
       extendedInfo: {
         formatter: Formatter.SUPPORTED_FORMATS.URL_LIST,
         value: insecureRecords
+      },
+      details: {
```

Comment:
> For example, pwmetrics, lighthouse-plots are consuming data that's in extendedInfo

> with no structure guarantees

sounds like a good way to break on every release...

> how about datagrid where the first column is a URL? URL will likely need its own render as it'll get forked for devtools revealing.

wouldn't a column of type URL work in that case? That's how the table formatter works now.

> or when URL is only sometimes a URL like in stylesheets that are inline and have previews

this is more of an issue, but still not clear why it can't be handled in the report, e.g. the element could default to the column type unless it specifies its own. Or whatever we come up with when we get to it. Anything to avoid `{type: 'text', text: 'my text'}` everywhere to enable an arbitrary nesting feature we'll rarely use anywhere.

(I also doubt that longterm our CSS will allow arbitrary nesting of components, so it does seem like the DOM generation code should mirror that)